ssl/test/runner/runner.go

```diff
@@ -19571,6 +19567,52 @@ func checkTests() {
 	}
 }
 
+func cipherSuitesContains(cipherSuites []uint16, id uint16) bool {
```

Comment:
nit: I'd call this `uint16SliceContains`. Just so that someone doesn't implement another `[]uint16` search in the future.